src/app/garfile/manifest/index.njk

```diff
@@ -78,131 +76,131 @@
+                  <div class="panel panel-border-narrow" id="details-content-{{ loop.index }}" aria-hidden="true">
+                    <table class="govuk-table" width="100%">
+                      <tbody class="govuk-table__body">
+                        <tr class="govuk-table__row">
+                          <td class="govuk-table__cell">{{__('field_gender')}}</td>
+                          <td class="govuk-table__cell">{{person.gender}}</td>
+                        </tr>
+                        <tr class="govuk-table__row">
+                          <td class="govuk-table__cell">{{__('field_birth_place')}}</td>
+                          <td class="govuk-table__cell">{{person.placeOfBirth}}</td>
+                        </tr>
+                        <tr class="govuk-table__row">
+                          <td class="govuk-table__cell">{{__('field_travel_document_type')}}</td>
+                          <td class="govuk-table__cell">{{person.documentType}}</td>
+                        </tr>
+                        <tr class="govuk-table__row">
+                          <td class="govuk-table__cell">{{__('field_travel_document_issuing_state')}}</td>
+                          <td class="govuk-table__cell">{{person.issuingState | upper }}</td>
+                        </tr>
+                        <tr class="govuk-table__row">
+                          <td class="govuk-table__cell">{{__('field_travel_document_expiry_date')}}</td>
+                          <td class="govuk-table__cell">{{person.documentExpiryDate}}</td>
+                        </tr>
+                        <tr class="govuk-table__row">
+                          <td class="govuk-table__cell" colspan="7">
+                            <button id="delete-person-{{ loop.index }}" class="link btn-link" name="deletePersonId" value="{{person.garPeopleId}}">{{__('table_delete')}}</button>
+                          </td>
+                        </tr>
+                      </tbody>
+                    </table>
+                  </div>
+                </details>
+              </div>
+            </td>
+          </tr>
+          {% endfor %}
+              </div>
+            </td>
+          </tr>
+        {% else %}
+          <tr class="govuk-table__row">
+            <td colspan="4"><p class="govuk-inset-text">{{__('manifest_no_records')}}</p></td>
+          </tr>
+        {% endif %}
+        </tbody>
+      </table>
+    </div>
+    <div class="govuk-grid-column-two-thirds">
+      <div class="govuk-form-group">
+        <label class="govuk-label" for="totalPersonsOnBoard">
+          {{__('total_persons_on_board')}}
+        </label>
 
-					<input class="govuk-input govuk-input--width-3" id="totalPersonsOnBoard" name="name" type="number" value={{manifest.items.length}}>
-				</div>
+        <input class="govuk-input govuk-input--width-3" id="totalPersonsOnBoard" name="name" readonly type="number" value="{{ manifest.items.length }}">
```

Comment:
Not sure why the display of the total is actually an input, maybe some screen reader thing?

It is set as readonly, because it kind of is, not sure if this affects accessibility.